src/operation-node/operator-node.ts

```diff
@@ -1,8 +1,7 @@
-import { ArrayItemType } from '../util/type-utils.js'
 import { freeze } from '../util/object-utils.js'
 import { OperationNode } from './operation-node.js'
 
-export const OPERATORS = [
+export const BINARY_OPERATORS = [
```

Comment:
This is actually a list of comparison operators since this is used with in `where`, `having` and `on`. Other operators like `+` should never be added to this list. A better name is probably `COMPARISON_OPERATORS`.

Reply:
That's better, agreed. 💯